node.js

```diff
@@ -258,5 +265,24 @@ module.exports = {
   clearCaches: function clearCaches () {
     filenessCache = { }
     configCache = { }
+  },
+
+  testCanIUse: function testCanIUse (latestVersion) {
+    var now = Date.now()
+    var digitsDiff = String(now).length - String(latestVersion).length
+    if (digitsDiff) {
+      latestVersion = padToEnd(latestVersion, digitsDiff)
+    }
+    var diffDates = now - latestVersion
+    diffDates = diffDates / 1000 / 60 / 60
+    var daysFromUpdate = Math.floor(diffDates / 24)
+    if (daysFromUpdate > DAYS_TO_UPDATE_CANIUSE) {
+      var pathToYarnLock = path.normalize('./yarn.lock')
```

Comment:
Not correct way to test `yarn.lock`. What if developer call Browserslist in project subdir? Look how we find `package.json`